java-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/resolution/SymbolSolverWithJavassistClassTest.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2016 Federico Tomassetti
```

Comment:
We should change that, but I will change it for all files to something like
`Copyright 2017 JavaParser team`